model-optimizer/extensions/middle/EltwiseInputReshape.py

```diff
@@ -73,50 +74,38 @@ def run_after(self):
         return [MiddleStart]
 
     def find_and_replace_pattern(self, graph: Graph):
-        for node in graph.get_data_nodes():
-            # Get all requested shapes for current node
-            # This mapping will contain pairs like {shape:[list of consumers nodes]}
-            mapping = {}
-            for consumer in node.out_nodes():
-                edge_attrs = graph.get_edge_data(node.id, consumer.id)[0]
-                if 'new_shape' in edge_attrs:
-                    if np.array_equal(edge_attrs['new_shape'], node.shape):
-                        continue
-                    new_shape = tuple([x for x in edge_attrs['new_shape']])
-                    if not new_shape in mapping:
-                        mapping.update({new_shape: [consumer]})
-                    else:
-                        mapping[new_shape].append(consumer)
+        for node in graph.get_op_nodes():
```

Comment:
Marking a node edge with 'new_shape' attribute in one transformation then calling directly this one to insert Reshape looks awkward. Also even a new code looks ugly. So I have a revolutionary suggestion. Let's merge `EltwiseInputReshape` and `EltwiseInputNormalize` into one transformation (keep `EltwiseInputReshape`) and let's insert `Reshape` operations "on the fly" when we detect that it is needed. It will make code much smaller + we will avoid redundant graph traversals and checking of all graph edges attributes (which is also time consuming).

Reply:
Done.